modules/draco/src/lib/draco-parser.ts

```diff
@@ -47,6 +46,7 @@ export type DracoParseOptions = {
   octahedronAttributes?: ('POSITION' | 'NORMAL' | 'COLOR' | 'TEX_COORD' | 'GENERIC')[];
 };
 
+// eslint-disable-next-line
```

Comment:
Specify, what exactly rule you are disabling.

Reply:
Yes let's be specific